tools/jdk/BUILD.java_tools

```diff
@@ -158,6 +158,45 @@ java_toolchain(
     target_version = "14",
 )
 
+# A toolchain that targets java 15.
+java_toolchain(
+    name = "toolchain_jdk_15",
```

Comment:
This could be combined in one single java_toolchain definition, together with "toolchain_jdk_14". Note that this was already the case for "toolchain_jdk_11" and "toolchain_jdk_12", in previous version of this file, but "toolchain_jdk_12" was  removed recently, and "toolchain_jdk_14 was very different compared to toolchain_jdk_11, so that reviewers suggested to split "toolchain_jdk_14" in own rule. That can be re-joined again with "toolchain_jdk_15".

Reply:
I don't think I can do this, because of the source_version/target_version attributes.  I could do a list comprehension though if that would make life cleaner?

Reply:
davido@ This was possible because of multiple releases of Java tools and substitution of JAVA_LANGUAGE_LEVEL. To ease releases of java_tools I prefer to have one version and toolchain_jdk_15 duplicated as it is.

There are other changes coming to make Java use toolchain resolution, where this will probably be cleaned up.